### PR TITLE
Updates to the legacy vpat

### PR DESCRIPTION
**Pull request by nickcall**
Bringing together the legacy and AIP vpats to form one document for all "legacy" sites across nature.com

### CODE REVIEW

File: nature-legacy.md [hunk 1]
```diff
@@ -137,9 +137,9 @@ Note to 1194.22: The Board interprets paragraphs (a) through (k) of this section
 | Criteria | Supporting features | Remarks and explanations |
 | -------- | ------------------- | ------------------------ |
 |(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided.|Supports with Exceptions|See [1194.22](#section-119422-web-based-internet-information-and-applications)|
-|(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided.|Supports|Markup is written with support for assistive technology.<br>Text is resizable by user. Website does not impede user from zooming into page, or from increasing the font size via their browser settings.|
```
Comment (alexkilgour) [inline nature-legacy.md]: is this no longer true?

Comment (nickcall) [reply nature-legacy.md]: As we say we support that we no longer need to give a description of why.

Comment (alexkilgour) [reply nature-legacy.md]: ah right, yeah i see

**Comment by alexkilgour** [comment]
This looks good to me. Might want @dotcode to take a look?

**Pull request merged**